src/pykeen/losses.py

```diff
@@ -12,7 +12,9 @@
 loss functions, we will assume the set of entities $\mathcal{E}$, set of relations $\mathcal{R}$, set of possible
 triples $\mathcal{T} = \mathcal{E} \times \mathcal{R} \times \mathcal{E}$, set of possible subsets of possible triples
 $2^{\mathcal{T}}$ (i.e., the power set of $\mathcal{T}$), set of positive triples $\mathcal{K}$, set of negative
-triples $\mathcal{\bar{K}}$, and scoring function (e.g., TransE) $f: \mathcal{T} \rightarrow \mathbb{R}$.
+triples $\mathcal{\bar{K}}$, scoring function (e.g., TransE) $f: \mathcal{T} \rightarrow \mathbb{R}$ and labeling
+function $l:\mathcal{T} \rightarrow \{0,1\}$ where a value of 1 denotes the triple is positive (i.e., $(h,r,t) \in
+\mathcal{K}$) and a value of 0 denotes the triple is negative (i.e., $(h,r,t) \notin \mathcal{K}$).
```

Comment:
following OWA, it is not negative, but just not positive :wink:

Reply:
Except for point-wise losses, we do not necessarily need this triple-label labeling, e.g. for pair-wise losses it suffices to have labels on a per-pair basis, i.e. triple 1 is "more positive" than triple 2

Reply:
I will work on re-wording :)

Reply:
Done in 9169ff25